### DISCUSSION TITLE
Update to Notes section

### DISCUSSION THREAD

**Discussion by steRnbear87** [tteck/Proxmox#3303] [Ideas]
Edit the notes section to use the app logo and also, if possible, a link to access the app.

This is what I use, just to give an idea

`<div align='center'><a href='https://Helper-Scripts.com' target='_blank' rel='noopener noreferrer'><img src='https://raw.githubusercontent.com/steRnbear87/dashboard-icons/main/120x120/radarr.png'/></a>

  # Radarr LXC

  ## <a href='http://IP_ADDRESS:7878' target='_blank'>http://IP_ADDRESS:7878</a>

  ## <a href='https://ko-fi.com/D1D7EP4GF'><img src='https://img.shields.io/badge/&#x2615;-Buy me a coffee-blue' /></a>

</div>
`

**Comment by alexanderjerome** [comment]
I proposed a similar idea in #3280 , +1

**Comment by tteck** [comment]
This is what sets the description in each app

https://github.com/tteck/Proxmox/blob/6e6a86aaa9d134d7e94efee9fff62c2cfea0b11c/misc/build.func#L640-L652

How would you propose modifying this to display each app's logo without rewriting the entire codebase?
